packages/api/emails/verifyHcEmail/html.pug

```diff
@@ -1,13 +1,13 @@
 div.main
-  img(src=`${imgUrl}/email/header.jpg` alt='header' style='width:100%; height:auto;')
+  img(src= `${imgUrl}/email/header.jpg` alt='header' style='width:100%; height:auto;')
 
   //- healthcare provider's first name
   p Hi #{name},
 
   p In order to process your application, we need you to verify your email. Please click the following link or copy and paste into your browser to complete the email verification:
 
   p VERIFICATION LINK: 
-    a(href=`${urlLink}`)
+    a(href=`${urlLink}`) clickhere
```

Comment:
Maybe it'd look better if we wrote this as two words (and maybe capitalized one or both of them)? Product might have some input there.

Reply:
I opted for 'Click here', I'll see what product has to say also.